eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Compiler/AsseticPass.php

```diff
@@ -25,6 +25,7 @@ public function process(ContainerBuilder $container)
 
         $assetFactoryDef = $container->findDefinition('assetic.asset_factory');
         $assetFactoryDef
+            ->setClass('eZ\Bundle\EzPublishCoreBundle\Assetic\AssetFactory')
```

Comment:
Would you care explaining why we need to _add_ this `setClass()` call ? I would have understood if we had replaced a `setClass('%something.class%')` or something related to the commit message, but not this.